src/cmap/connection.ts

```diff
@@ -309,11 +311,18 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
     this[kLastUseTime] = now();
```

Comment:
we now have cleanup logic scattered across four different functions in the connection class.  Is there any reason we can't rely on `destroy({ force: true})` in all three helpers?